plutus-benchmark/validation/Main.hs

```diff
@@ -157,7 +152,12 @@ main :: IO ()
 main = do
   cfg <- getConfig 20.0  -- Run each benchmark for at least 20 seconds.  Change this with -L or --timeout (longer is better).
   options <- execParser $ parserInfo cfg
-  benchmarks <- if quick options
-                then mapM mkContractBMs contractDirs2
-                else mapM mkContractBMs contractDirs
+  scriptDirectory <- getScriptDirectory
+  files0 <- listDirectory scriptDirectory  -- Just the filenames, not the full paths
+  let files1 = naturalSort $ filter (isExtensionOf ".flat") files0  -- Just in case there's anything else in the directory.
+               -- naturalSort puts the filenames in a better order than Data.List.Sort
```

Comment:
controversial suggestion: I don't care. The only thing we usually care about is the diffs!

Reply:
Yeah, but it's really annoying when they appear out of order!!  We didn't get that before because I manually put extra zeros in the problematic filenames.

Reply:
have you tried......... not caring :trollface: 

anyway, do what you like!

Reply:
I tried not to care but failed.  It's a kind of therapeutic exercise to do it properly though.